pkg/types/block.go

```diff
@@ -0,0 +1,29 @@
+package types
+
+import (
+	"github.com/pkg/errors"
+)
+
+func (m *Block) GetTxIDs() (*BlockTxIds, error) {
+	txIDs := make([]string, 0)
+	switch m.Payload.(type) {
+	case *Block_DataTxEnvelopes:
+		dataTxs := m.GetDataTxEnvelopes().Envelopes
+		for _, tx := range dataTxs {
+			txIDs = append(txIDs, tx.Payload.TxId)
+		}
+
+	case *Block_ConfigTxEnvelope:
+		txIDs = append(txIDs, m.GetConfigTxEnvelope().Payload.TxId)
+
+	case *Block_DbAdministrationTxEnvelope:
+		txIDs = append(txIDs, m.GetDbAdministrationTxEnvelope().Payload.TxId)
+
+	case *Block_UserAdministrationTxEnvelope:
+		txIDs = append(txIDs, m.GetUserAdministrationTxEnvelope().Payload.TxId)
+
+	default:
+		return nil, errors.Errorf("unknown block payload")
+	}
+	return &BlockTxIds{TxId: txIDs}, nil
+}
```

Comment:
There is already a block to txIDs in util pkg. https://github.com/hyperledger-labs/orion-server/blob/main/internal/httputils/blocks.go

Reply:
I missed it, will change